lib/matplotlib/font_manager.py

```diff
@@ -1417,11 +1473,14 @@ def _get_font(filename, hinting_factor, *, _kerning_factor, thread_id):
 def get_font(filename, hinting_factor=None):
     # Resolving the path avoids embedding the font twice in pdf/ps output if a
     # single font is selected using two different relative paths.
-    filename = _cached_realpath(filename)
+    if isinstance(filename, FontsInterface):
+        paths = tuple([_cached_realpath(fname) for fname in filename.get_filepaths()])
```

Comment:
You can remove the brackets and will likely even improve the performance ever so slightly: `[f(x) for x in y]` creates a list, `(f(x) for x in y)` a generator that you then feed to the `tuple` function.

Reply:
makes sense! 👍🏼